test/configs/AuthenticatedFileResourceStoreConfig.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  AcceptPreferenceParser,
+  AclManager,
+  AuthenticatedLdpHandler,
+  BasicRequestParser,
+  BasicResponseWriter,
+  BasicTargetExtractor,
+  CompositeAsyncHandler,
+  HttpHandler,
+  InteractionController,
+  MetadataController,
+  Operation,
+  QuadToTurtleConverter,
+  RepresentationConvertingStore,
+  ResourceStore,
+  ResponseDescription,
+  RuntimeConfig,
+  TurtleToQuadConverter,
+  UrlBasedAclManager,
+  WebAclAuthorizer,
+} from '../..';
+import { UnsecureWebIdExtractor } from '../../src/authentication/UnsecureWebIdExtractor';
+import { RawBodyParser } from '../../src/ldp/http/RawBodyParser';
+import { DeleteOperationHandler } from '../../src/ldp/operations/DeleteOperationHandler';
+import { GetOperationHandler } from '../../src/ldp/operations/GetOperationHandler';
+import { PostOperationHandler } from '../../src/ldp/operations/PostOperationHandler';
+import { PutOperationHandler } from '../../src/ldp/operations/PutOperationHandler';
+import { MethodPermissionsExtractor } from '../../src/ldp/permissions/MethodPermissionsExtractor';
+import { FileResourceStore } from '../../src/storage/FileResourceStore';
+import { UrlContainerManager } from '../../src/storage/UrlContainerManager';
+import { ServerConfig } from '../configs/ServerConfig';
+
+// This is the configuration from bin/server.ts
+
+export class AuthenticatedFileResourceStoreConfig implements ServerConfig {
+  public store: ResourceStore;
+  public aclManager: AclManager;
+  public runtimeConfig: RuntimeConfig;
+
+  public constructor() {
+    this.runtimeConfig = new RuntimeConfig({
+      base: 'http://test.com',
+      rootFilepath: 'uploads/',
+    });
+
+    const fileStore = new FileResourceStore(
+      this.runtimeConfig,
+      new InteractionController(),
+      new MetadataController(),
+    );
+
+    const converter = new CompositeAsyncHandler([
+      new QuadToTurtleConverter(),
+      new TurtleToQuadConverter(),
```

Comment:
Just changing them does not work for the tests, freya says that the problem is in the `QuadToRdfConverter`. The integrationtests of `AuthenticatedLdpHandler` are not updated so that's why the test did not fail. When running the test no error occurs but nothing is returned.

Reply:
The only reason I can think of that the old one would work while the new one wouldn't is that no `Accept` header was passed along (but then again I think the converter wouldn't even be called in either case), so we definitely should find out what the issue is. There is an issue to remove the old ones (#103) so they will need to go anyway.

Reply:
Found the issue, the data is in _getBuffer(). Might have to do with the fact that there's no more piping to `new StreamWriter({ format: TEXT_TURTLE })`. But if it's no problem that it gets into _getBuffer() than this issue is fixed.

Reply:
Yea that's fine. I'm not sure myself what exactly the differences are between getBuffer and getdata from that library (the second one maybe only works when the stream is in object mode?). From the HTTP side this is not going to make a difference.